client/src/ui/molecules/menu/index.scss

```diff
@@ -1,8 +1,12 @@
 .top-level-entry-container {
-  &.active > a {
+  &:not(.active):not(.user-menu):not(:hover) {
+    opacity: 0.775;
```

Comment:
Why are we using opacity rather than setting `color`?

Using opacity breaks Firefox's accessibility inspector, which doesn't take it into account when assessing contrast:

![image](https://github.com/mdn/yari/assets/755354/b281ca78-192d-41e4-9920-533ec3e55872)

Reply:
I used `opacity` for simplicity, because it avoids having to define two additional colors for light and dark theme, and because it also applies to the "New" pill on the Tools menu (which would require another two colors).

TIL about Firefox's accessibility inspector ignoring opacity, but technically this PR increases contrast.

Reply:
@LeoMcA Are you okay with merging this?

Reply:
> it also applies to the "New" pill on the Tools menu (which would require another two colors)

Why does it apply the the "New" pill? Isn't the point of that pill to draw attention? Why are we then making it stand out less?

> but technically this PR increases contrast.

Does it? It decreases the contrast of the majority of links in the header. Taking a screenshot and colour picking the colour (`#757575`), then putting it in https://webaim.org/resources/contrastchecker/ shows we fail AAA contrast, which I don't think is good enough for some of the most prominent text on our site:

![image](https://github.com/user-attachments/assets/4f307b71-c69c-465a-85ed-87840302b455)